src/allmydata/immutable/filenode.py

```diff
@@ -139,7 +139,7 @@ def _gather_repair_results(self, ur, cr, crr):
             for server in servers:
                 sm.add(shnum, server)
                 servers_responding.add(server)
-        servers_responding = sorted(servers_responding)
+        servers_responding = sorted(servers_responding, key=lambda o: id(o))
```

Comment:
Not sure why the sort was here to begin with... objects in Python 3 aren't sortable if they're not explicitly comparable, and these aren't. Other option is just get rid of the sorting.

Reply:
I can't tell why this is sorted either.
I'd be fine with removing it if the tests still pass (perhaps it was an attempt to make testing easier/more-predictable? if it affects output order somehow?)

That said, there are certainly cases where the order matters for servers .. but I don't think this is one (and if it were, they should be sorted on their "server key" or something, not where they landed in memory).